docs/tutorials/09_properties.ipynb

```diff
@@ -0,0 +1,219 @@
+    "print(base_problem.properties)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d76c0f7",
+   "metadata": {},
+   "source": [
+    "You can now modify the contents of the `PropertiesContainer` to your desire:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "fb1d2956",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AngularMomentum is in problem.properties: False\n",
+      "Adding AngularMomentum to problem.properties...\n",
+      "AngularMomentum is in problem.properties: True\n",
+      "Discarding AngularMomentum from problem.properties...\n",
+      "AngularMomentum is in problem.properties: False\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.properties import AngularMomentum\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)\n",
+    "\n",
+    "print(\"Adding AngularMomentum to problem.properties...\")\n",
+    "base_problem.properties.add(AngularMomentum(2))\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)\n",
+    "\n",
+    "print(\"Discarding AngularMomentum from problem.properties...\")\n",
+    "base_problem.properties.discard(AngularMomentum)\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b72239b",
+   "metadata": {},
+   "source": [
+    "The specific subclasses of `BaseProblem` for the various stacks such as [electronic structure](01_electronic_structure.ipynb), [vibrational structure](02_vibrational_structure.ipynb) or [lattice models](10_lattice_models.ipynb) make things even simpler for you, because they provide direct attributes for the built-in properties of Qiskit Nature:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6c251832",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<qiskit_nature.second_q.properties.particle_number.ParticleNumber object at 0x14bfa1aa5b80>\n",
+      "<qiskit_nature.second_q.properties.angular_momentum.AngularMomentum object at 0x14bfa1aa5a60>\n",
+      "<qiskit_nature.second_q.properties.magnetization.Magnetization object at 0x14bfa1aa5ac0>\n",
+      "<qiskit_nature.second_q.properties.dipole_moment.ElectronicDipoleMoment object at 0x14bfa1aa5f70>\n",
+      "None\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.drivers import PySCFDriver\n",
+    "\n",
+    "es_problem = PySCFDriver().run()\n",
+    "\n",
+    "print(es_problem.properties.particle_number)\n",
+    "print(es_problem.properties.angular_momentum)\n",
+    "print(es_problem.properties.magnetization)\n",
+    "print(es_problem.properties.electronic_dipole_moment)\n",
```

Comment:
In the QCSchema tutorial it mentioned the include dipole setting - should we note that via the way the driver is run here that this is set which is why the dipole moment exists here above

Reply:
Yes indeed. The `ElectronicStructureDriver.run()` method does not expose the `include_dipole` setting so it will always default to `True`

One would need to run the driver more manually to disable that and this depends on the driver, for example:
```python
driver = PySCFDriver()
driver.run_pyscf()
problem = driver.to_problem(include_dipole=False)
```

Reply:
I've added a comment to the tutorial :+1: